services/chaindb/postgresql/validators.go

```diff
@@ -279,13 +279,12 @@ func (s *Service) ValidatorsByIndex(ctx context.Context, indices []phase0.Valida
 	return validators, nil
 }
 
-// ValidatorBalancesByIndexAndEpoch fetches the validator balances for the given validators and epoch.
```

Comment:
`ValidatorBalancesByIndexAndEpoch` should remain as a function, as it is public and may be used by other codebases when fetching information from the chain database.  Please retain this function, and create a new function for `ValidatorBalancesByEpoch`.